contracts/PartnerFund.sol

```diff
@@ -194,7 +194,7 @@ contract PartnerFund is Ownable, Beneficiary, TransferControllerManageable {
     //
     // Deposit history retrieval functions
     // -----------------------------------------------------------------------------------------------------------------
-    function deposit(address tag, uint index) public view isRegisteredTag(tag) returns (int256 balance, address currencyCt, uint256 currencyId, uint256 blockNumber) {
+    function deposit(address tag, uint index) public view isRegisteredTag(tag) returns (int256 balance, uint256 blockNumber, address currencyCt, uint256 currencyId) {
```

Comment:
Correlating this function with `depositFromAddress()` below and `deposit()` from _SecurityBond_ I suggest changing the signature slightly. In _ClientFund_, _Beneficiary_ and elsewhere the deposit function parameters first have amount followed currency contract and currency ID. I suggest a similar pattern for return parameters for the 3 altered deposit functions in this PR. E.g. `amount` (renamed from `balance`) followed by `currencyCt` and `currencyId`. Then the temporal return parameter can come either before `amount` or after `currencyId`.

Reply:
`ClientFund` and `SecurityBond` have the `amount/balance` followed by the `timestamp/blocknumber` and after they have the currency. That's why I changed the signature.

Reply:
Right. I suggest it will make more sense to have the temporal return parameter either before `amount` or after `currencyId`. This makes it more consistent with the functions that make the deposits. I'll defer my suggestions to a new issue to be created shortly.